jenkinsbox.yml

```diff
@@ -71,6 +64,11 @@
       - apache
 
   # @tag environment.
+  - name: Set Jenkins server IP
+    set_fact:
+      jenkins_server_ip: "{{ ansible_default_ipv4.address }}"
```

Comment:
@podarok I think we can pass one additional variable into `extra_vars` in bash script and solve this https://github.com/propeoplemd/cibox/issues/326